qupath-extension-openslide/src/main/java/qupath/lib/images/servers/openslide/OpenSlideServerBuilder.java

```diff
@@ -45,9 +45,9 @@
  * @author Pete Bankhead
  *
  */
-public class OpenslideServerBuilder implements ImageServerBuilder<BufferedImage> {
+public class OpenSlideServerBuilder implements ImageServerBuilder<BufferedImage> {
```

Comment:
I see the reasoning, but since the class name is serialized in the project I think there's a strong chance this will break compatibility.

Breaking compatibility is sometimes necessary - especially for scripts / analysis, where a clean break is better than a subtle inconsistency - but if at all possible we'd want projects, images and objects to open properly in new versions. And, if saving the data, not to break any attempt to reopen the same data in an old QuPath version.

Will definitely need to check for that anyway. It might be possible to change the class, but ensure whatever is serialized is still interpretable by OpenSlide (e.g. using `"providerClassName": "openslide"`).

Reply:
Ah, won't bother then, it's just aesthetic

Reply:
I'm in favor of the aesthetics! Just flagging it's probably more awkward

Reply:
I'm in favour of aesthetics generally, but not really at the risk of breaking changes

Reply:
This is the offending bit of code: 
https://github.com/qupath/qupath/blob/533ce0cf50ca2a5627b55a8c1fe351cdc72e0694/qupath-core/src/main/java/qupath/lib/images/servers/ImageServerBuilder.java#L348-L350

Could modified it to give a bit more wiggle room to make changes in a future update (since a v0.6 project that doesn't work in v0.4 doesn't seem as bad as one that doesn't work in v0.5)